cmd/argocd/commands/admin/generatespec_utils_test.go

```diff
@@ -0,0 +1,38 @@
+package admin
+
```

Comment:
Wonder if you could please add a few test cases for PrintResources? For example, there are multiple resources to be printed.